packages/email-auth/src/email-auth-server.ts

```diff
@@ -105,30 +103,35 @@ export class EmailAuthServer {
               },
             }),
           );
-
+  
           return;
         } else {
-          await this.storageAdapter.deleteOtp({
+          await storageAdapter.deleteOtp({
             otp: body.otp,
           });
         }
-
-        const token = jsonwebtoken.sign({}, this.opts?.jwtSecret as string, {
-          expiresIn: this.opts?.jwtTTL as string,
+  
+        const token = jsonwebtoken.sign({}, opts?.jwtSecret as string, {
+          expiresIn: opts?.jwtTTL as string,
           subject: user.emailAddress,
         });
-   
-        res.writeHead(200, { 'Content-Type': 'application/json' })
+  
+        res.writeHead(200, { 'Content-Type': 'application/json' });
```

Comment:
use express functions such as `res.send` and `res.set`

Reply:
fixed